native/cocos/engine/Engine.cpp

```diff
@@ -369,7 +371,8 @@ bool Engine::dispatchWindowEvent(const WindowEvent &ev) {
     } else if (ev.type == WindowEvent::Type::SIZE_CHANGED ||
                ev.type == WindowEvent::Type::RESIZED) {
         cc::EventDispatcher::dispatchResizeEvent(ev.width, ev.height);
-        auto *w = CC_GET_PLATFORM_INTERFACE(ISystemWindow);
+        auto *w = CC_GET_SYSTEM_WINDOW(ev.windowId);
+        CC_ASSERT(w && "MainSystemWindow is null.");
```

Comment:
`w && "MainSystemWindow is null."` will always be true. Just `CC_ASSERT(w)` is enough.